src/stackedtime/sim_nr.jl

```diff
@@ -78,3 +107,23 @@ Solve the simulation problem.
     return false
 end
 
+"""
+
+"""
+function schedule_λ_default(λ::Float64, it::Int, nFx::Float64, Δx::AbstractArray{Float64})
+    nΔx = λ * norm(vec(Δx), Inf)
+    # Initialize λ
+    if it == 1 && nFx >= 0.5
+        λ = 0.1250
+    end
+    # Set Δλ
+    (nFx > 1.) ? (Δλ = 0.05) : (Δλ = 0.15)
```

Comment:
This line can be put inside the `if normDeltax` block since we don't use delta_lambda when the if statement evaluates to false.